recipe/meta.yaml

```
@@ -5,12 +5,13 @@ source:
   git_url: https://github.com/GenericMappingTools/gmt.git
   git_rev: 07769944b445953a7a203f20657285d5a18c5e56
 build:
-  number: 1
+  number: 2
   skip: true  # [win and vc<14]
```

Skip ppc64le here.

```suggestion
  skip: true  # [(win and vc<14) or ppc64le]
```

By the way, do you know if we can remove that `win and vc<14` line?

---

I'm not sure. These lines were added five years ago (14a9df63e46fa5454595c0006ac1ab6399f286cf in PR #62). So maybe we can remove it and see if everything is fine.

---

Yeah, let's try and remove it
```suggestion
  skip: true  # [ppc64le]
```

---

Builds are successful!